docs/developers/operations-api/data-management.md

```diff
@@ -0,0 +1,98 @@
+# Data Management
```

Comment:
Can we move these operations to bulk-operations.md? (I think they make sense there?)

Reply:
Sure :)